categorization/bin/P_id_S_id_Age_sex.py

```diff
@@ -0,0 +1,76 @@
+from typing import Dict
+
+from langchain_community.chat_models import ChatOllama
+from langchain_core.output_parsers import JsonOutputParser
+from langchain_core.prompts import PromptTemplate
+from langchain_core.pydantic_v1 import BaseModel, Field
+
+def Categorize1(result_dict: Dict[str, str]) -> None:
+
+     # Initialize model
+    llm = ChatOllama(model="gemma")
+
+    # # Create output structure
+    # class ColumnCategorization(BaseModel):
+    #     predictedColumn: str = Field(description="Column name predicted by the LLM within the NB data model entities")
+
+    # # Set up parser
+    # parser = JsonOutputParser(pydantic_object=ColumnCategorization)
+
+    # Create prompt template
+    prompt = PromptTemplate(
+        template='''Given the column data {column}: {content}, determine the category and give only the category name as output
+
+Examples:
+1. Input: "participant_id: sub-01 sub-02 sub-03"
+   Output: Subject_IDs
+
+2. Input: 'pheno_age: ["34,1", "35,3", "NA", "39,0", "22,1", "23,2", "21,1", "22,3", "42,5", "43,2"]'
+   Output: Age
+
+3. Input: "session_id: ses-01 ses-02"
+   Output: Session_IDs
+
+4. Input: "pheno_sex : ["F", "F", "M", "M", "missing", "missing", "F", "F", "M", "M"]" 
+   Output: Sex
```

Comment:
Here I think it would be nice (in general for categorical variables) to first hand over the column for categorization of the column itself and in a second step (I think it can be nicely done with langchain) make sense about the levels contained in the column (I guess if there are only numbers present we won't have a chance to identfy the columns correctly, but e.g., with M/F or something like that this might succeed).

Reply:
Um yess Thanks!!
I thought the numbers are restricted to 0 1 2...
Yes we can surely do that

Reply:
I am creating another PR for Age and Sex So resolving this conversion here for now will include this in the code for sex